Makefile

```diff
@@ -245,7 +248,7 @@ $(BUILD_DIRECTORY)/$(APPLICATION_NAME)-cli-$(TARGET_PLATFORM)-$(APPLICATION_VERS
 	mkdir $@
 	cp $(word 1,$^) $@
 	cp $(word 2,$^) $@
```

Comment:
Based on the CI failures, I wonder if this might need to be using `$(CPRF)` too?